helpers/file/file_type.go

```diff
@@ -1,12 +1,12 @@
-package mt
```

Comment:
In general, smartling should support the same set of file types for File API and MT File API. However, in reality, the second API is behind and supports fewer file types.
I'm fine to reuse enum for both APIs (actually 3 APIs: File, Job Batches, MT File). So it is up to you how to organize enum.